### ISSUE TITLE
add support accessing feeds and content over IPFS

### ISSUE THREAD

**Issue by n8fr8** [Automattic/pocket-casts-android#429]
### What problem does this address?

Enabling people to access podcasts published through the decentralized web will allow more resilient and free access to podcast content, and provide more options for podcast hosting. Podcasts could even be hosted and accessed from any computer that can run a local IPFS daemon, or using new platforms like web3.storage. 

### What is your proposed solution?

1) Implement access to IPFS via gateways, as Opera on Android has done:
https://blog.ipfs.tech/2020-03-30-ipfs-in-opera-for-android/

2) in the future, add the option for accessing IPFS via a full local IPFS node running on the device for true decentralized access

### Did you search for existing feature suggestions?

- [X] I have searched for existing feature suggestions.

**Comment by chris122380** [comment]
Are there IPFS specific podcasts? This makes sense for browser but not for a podcast app if there aren't any podcasts using IPFS.

**Comment by n8fr8** [comment]
True, this is a bit of the cart before the horse. That said, if there is an interest in showing leadership in the decentralized web space, this would be a way to do it! Storing and distributing podcasts (RSS+MP3) on IPFS makes a great deal of sense, considering they are static content.
